aws/table_aws_efs_file_system.go

```diff
@@ -0,0 +1,256 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "owner_id",
+				Description: "The AWS account that created the file system.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_token",
+				Description: "The opaque string specified in the request.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_time",
+				Description: "The time that the file system was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "life_cycle_state",
+				Description: "The lifecycle phase of the file system.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "number_of_mount_targets",
+				Description: "The current number of mount targets that the file system has.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "size_in_bytes",
+				Description: "The latest known metered size (in bytes) of data stored in the file system.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "performance_mode",
+				Description: "The performance mode of the file system.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "encrypted",
+				Description: "A Boolean value that, if true, indicates that the file system is encrypted.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "kms_key_id",
+				Description: "The ID of an AWS Key Management Service (AWS KMS) customer master key (CMK) that was used to protect the encrypted file system.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "throughput_mode",
+				Description: "The throughput mode for a file system.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "provisioned_throughput_in_mibps",
+				Description: "The throughput, measured in MiB/s, that you want to provision for a file system.",
+				Type:        proto.ColumnType_DOUBLE,
+			},
+			{
+				Name:        "policy",
+				Description: "The JSON formatted FileSystemPolicy for the EFS file system.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getElasticFileSystemPolicy,
+				Transform:   transform.FromField("Policy"),
+			},
+			{
+				Name:        "policy_std",
+				Description: "Contains the policy in a canonical form for easier searching.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getElasticFileSystemPolicy,
+				Transform:   transform.FromField("Policy").Transform(unescape).Transform(policyToCanonical),
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags associated with Filesystem.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("FileSystemId"),
```

Comment:
Set the `Name` as title, if not null.

Reply:
I think FileSystemId is better as title than name. The name can also be duplicate